Developers/Development_Setup.rst

```diff
@@ -28,6 +28,7 @@ repository. The former is the core of coala, and the latter contains the set
 of standard bears. You can fork and clone the repositories from:
```

Comment:
Sorry for my ignorance to the importance of periods in body and the short log. Won't happen next time. :+1: